config_parser.go

```diff
@@ -2,12 +2,17 @@ package main
 
 import (
 	"github.com/bitrise-io/go-steputils/stepconf"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bmatcuk/doublestar/v3"
```

Comment:
[we used](https://github.com/search?q=org%3Abitrise-steplib+glob&type=code) this glob package previously: github.com/ryanuber/go-glob.

Is there any reason to replace it?

Reply:
It doesn't seem to support file system globbing for the first sight. That's why I changed.